src/main/java/WayofTime/bloodmagic/command/sub/SubCommandOrb.java

```diff
@@ -1,107 +1,116 @@
 package WayofTime.bloodmagic.command.sub;
 
 import WayofTime.bloodmagic.core.data.SoulNetwork;
+import WayofTime.bloodmagic.core.registry.OrbRegistry;
+import WayofTime.bloodmagic.util.Utils;
 import WayofTime.bloodmagic.util.helper.NetworkHelper;
 import WayofTime.bloodmagic.util.helper.PlayerHelper;
-import WayofTime.bloodmagic.command.CommandBloodMagic;
-import WayofTime.bloodmagic.util.Utils;
-import WayofTime.bloodmagic.util.helper.TextHelper;
-import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
-import net.minecraft.command.PlayerNotFoundException;
-import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.server.MinecraftServer;
-import net.minecraft.util.text.TextComponentString;
-
-import java.util.Locale;
+import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraftforge.server.command.CommandTreeBase;
+import net.minecraftforge.server.command.CommandTreeHelp;
+
+public class SubCommandOrb extends CommandTreeBase {
+    public SubCommandOrb() {
+        addSubcommand(new Get());
+        addSubcommand(new Set());
+        addSubcommand(new CommandTreeHelp(this));
+    }
 
-public class SubCommandOrb extends CommandBase {
     @Override
     public String getName() {
         return "orb";
     }
 
     @Override
     public String getUsage(ICommandSender commandSender) {
-        return TextHelper.localizeEffect("commands.bloodmagic.orb.usage");
+        return new TextComponentTranslation("commands.bloodmagic.orb.usage").getFormattedText();
     }
 
     @Override
     public int getRequiredPermissionLevel() {
         return 2;
     }
 
-    @Override
-    public void execute(MinecraftServer server, ICommandSender commandSender, String[] args) throws CommandException {
-        if (args.length > 0) {
+    abstract class OrbCommand extends CommandTreeBase {
+        public String help = new TextComponentTranslation("commands.bloodmagic.orb." + getName() + ".help", getInfo()).getFormattedText();
```

Comment:
Again, you cannot pre-translate like this. Localization must be done on the fly.